documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-mqtt5.md

```diff
@@ -0,0 +1,185 @@
+---
+title: MQTT 5 protocol binding
+keywords: binding, protocol, mqtt
+tags: [protocol, connectivity]
+permalink: connectivity-protocol-bindings-mqtt.html
+---
+
+Consume messages from MQTT brokers via [sources](#source-format) and send messages to MQTT brokers via 
+[targets](#target-format).
+
+## Content-type
+
+When MQTT messages are sent in [Ditto Protocol](protocol-overview.html),
+the payload should be `UTF-8` encoded strings.
+
+If messages which are not in Ditto Protocol should be processed, a [payload mapping](connectivity-mapping.html) must
+be configured for the connection in order to transform the messages.
+
+## MQTT 5 properties
+
+Supported MQTT 5 properties which are interpreted in a specific way are:
+
+* `9 (0x09) Correlation Data`: For correlating request messages and events. Twin events have the correlation IDs of
+  [Twin commands](protocol-twinlive.html#twin) that produced them. Stored in the ditto protocol header `correlation-id`.
+* `8 (0x08) Response Topic`: The MQTT topic a requests response is expected in.
+  If a command sets the header `reply-to`, then its response is published at the topic equal to the header value.
+* `3 (0x03) Content Type`: The UTF-8 encoded string representation of the payloads content MIME type.
+
+## Specific connection configuration
+
+The common configuration for connections in [Connections > Sources](basic-connections.html#sources) and 
+[Connections > Targets](basic-connections.html#targets) applies here as well. 
+Following are some specifics for MQTT connections:
+
+### Source format
+
+For an MQTT connection:
+
+* Source `"addresses"` are MQTT topics to subscribe to. Wildcards `+` and `#` are allowed.
+* `"authorizationContext"` may _not_ contain placeholders `{%raw%}{{ header:<header-name> }}{%endraw%}` as MQTT 5
```

Comment:
That's not correct - for MQTT 5 we have the user properties which can be mapped to headers :)

Reply:
I removed the wrong statement and replaced it with a short description borrowed from amqp-1 docs.